Automattic/woocommerce-payments
# Refresh page when ECE is dismissed

Fixes #9794 

#### Changes proposed in this Pull Request

Refresh the page's shipping information when ECE is dismissed. This ensures that the shipping options on the cart and checkout pages are updated without requiring the user to reload the page.

#### Testing Instructions

Repeat the following steps for:

- Cart and Checkout pages.
- Shortcode and Blocks versions of these pages.
- Google Pay and Apple Pay.

1. Add products to your cart that require shipping.
2. Navigate to the page under test (cart or checkout).
3. Click on the express payment method you are testing (Apple Pay or Google Pay).
4. Select a shipping address in the payment sheet.
5. Close the payment sheet without completing the payment.
6. Verify that the shipping address and shipping rates on the page are updated to reflect the selected address.
7. Reload the page and confirm that the shipping information remains the same as in the previous step.

-------------------


**Post merge**
